INSTALL.md

```diff
@@ -71,12 +81,30 @@ sudo mysterium_server --data-dir=/var/lib/mysterium-node --config-dir=/etc/myste
  * https://github.com/MysteriumNetwork/node/releases/download/{VERSION}/mysterium-client_linux_amd64.deb
  * https://github.com/MysteriumNetwork/node/releases/download/{VERSION}/mysterium-client_linux_armhf.deb
 
+### Install latest OpenVPN and dependencies
+
+```bash
+apt-get update && apt-get install -y curl
+curl -s https://swupdate.openvpn.net/repos/repo-public.gpg | apt-key add && echo "deb http://build.openvpn.net/debian/openvpn/stable xenial main" > /etc/apt/sources.list.d/openvpn-aptrepo.list && rm -rf /var/cache/apt/* /var/lib/apt/lists/*
+apt-get update
+apt-get install openvpn resolvconf ca-certificates iproute2 sudo
+```
+
 ### Installation
 ```bash
 wget https://github.com/MysteriumNetwork/node/releases/download/{VERSION}/mysterium-client_linux_amd64.deb
 sudo dpkg --install --force-depends mysterium-client_linux_amd64.deb
 sudo apt-get install --fix-broken
 ```
+
+In order for mysterium client to change system DNS servers to ones provided by VPN node
+ it needs to modify /etc/resolv.conf
+This change is performed by resolvconf utility. After resolvconf utility is installed, you need:
+```bash
+dpkg-reconfigure resolvconf
```

Comment:
Yet another instruction :/
Is it possible to automate this installation code, like we do with tun devices?